src/elfpy/pricing_models/hyperdrive.py

```diff
@@ -278,9 +294,9 @@ def calc_out_given_in(
             user_result=user_result,
             market_result=market_result,
             breakdown=TradeBreakdown(
-                without_fee_or_slippage=float(flat + Decimal(curve.breakdown.without_fee_or_slippage)),
-                without_fee=float(flat + Decimal(curve.breakdown.without_fee)),
-                fee=curve.breakdown.fee,
-                with_fee=float(flat + Decimal(curve.breakdown.with_fee)),
+                without_fee_or_slippage=float(flat_without_fee + Decimal(curve.breakdown.without_fee_or_slippage)),
+                without_fee=float(flat_without_fee + Decimal(curve.breakdown.without_fee)),
+                fee=curve.breakdown.fee + float(redemption_fee),
```

Comment:
also should be `float(redemption_fee) + curve.breakdown.fee)`

Reply:
roger